scripts/parseJson.js

```diff
@@ -0,0 +1,171 @@
+const loadJsonFile = require('load-json-file');
+const writeJsonFile = require('write-json-file');
+
+loadJsonFile('./scripts/tempConfig.json').then(json => {
```

Comment:
I'd recommend you use `path` to define file paths, that way it's compatible with Mac & Windows also use `__dirname` to define the absolute filepath that way you can execute the script from any directory.

```js
path.join(__dirname, 'tempConfig.json')
```